R/apply_tfrmt.R

```diff
@@ -34,6 +34,27 @@ apply_tfrmt <- function(.data, tfrmt, mock = FALSE){
       fail_desc = "Failure while aligning data values"
     )
 
+if(is_empty(tfrmt$sorting_cols)==FALSE){
```

Comment:
Rather than adding this to the already rather long function that is apply_tfrmt can you break it out into its own function

Reply:
done